e2e/cypress/integration/search/search_group_message_spec.js

```diff
@@ -11,14 +11,15 @@ const groupMembers = ['aaron.peterson', 'aaron.ward', 'samuel.tucker'];
 
 describe('Search', () => {
     before(() => {
-        cy.apiLogin('user-1');
-        cy.apiSaveTeammateNameDisplayPreference('username');
+        cy.loginAsNewUser().then(() => {
```

Comment:
Login as new user instead of existing user-1 so that the new GM will be on the first list on search